### PR TITLE
add 3d tiles example

### PR DESCRIPTION
**Pull request by xintongxia**
color-normal-demo

### CODE REVIEW

File: modules/3d-tiles/src/classes/tile-3d-feature-table.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ export default class Tile3DFeatureTable {
 
     if (jsonValue && Number.isFinite(jsonValue.byteOffset)) {
       if ('componentType' in jsonValue) {
-        // componentType = ComponentDatatype.fromName(jsonValue.componentType);
```
Comment (ibgreen) [inline modules/3d-tiles/src/classes/tile-3d-feature-table.js]: Cut?

File: modules/3d-tiles/src/parsers/parse-3d-tile-point-cloud.js [hunk 2]
```diff
@@ -36,8 +36,6 @@ function extractPointCloud(tile) {
     throw new Error('POINTS_LENGTH must be defined');
   }
 
-  // const batchTable = new Tile3DBatchTable(tile);
```
Comment (ibgreen) [inline modules/3d-tiles/src/parsers/parse-3d-tile-point-cloud.js]: Point clouds do have batch tables: https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/specification/TileFormats/PointCloud#layout

We'll need to add this back soon.

File: modules/3d-tiles/src/parsers/parse-3d-tile-point-cloud.js [hunk 3]
```diff
@@ -50,6 +48,77 @@ function extractPointCloud(tile) {
   tile.isRGB565 = false;
   tile.isOctEncoded16P = false;
 
+  parsePositions(tile, featureTable);
```
Comment (ibgreen) [inline modules/3d-tiles/src/parsers/parse-3d-tile-point-cloud.js]: +1 this looks organized!

File: modules/3d-tiles/src/parsers/parse-3d-tile-point-cloud.js [hunk 4]
```diff
@@ -50,6 +48,77 @@ function extractPointCloud(tile) {
   tile.isRGB565 = false;
   tile.isOctEncoded16P = false;
```
Comment (ibgreen) [inline modules/3d-tiles/src/parsers/parse-3d-tile-point-cloud.js]: Maybe `Object.seal(this)` here to make sure we have initialized all values with defaults.

**Comment by xintongxia** [comment]
Will address the comments in following PR

**Pull request merged**